docs/changes/utils/16187.api.rst

```diff
@@ -0,0 +1 @@
+The return type of ``IERS_Auto.open()`` is no longer ``IERS_B`` when automatic updating of the IERS-A file is disabled or the downloading of the new file fails.
```

Comment:
I think we should add what it is now! Maybe 
```
IERS_Auto.open() now defaults to the combination of IERS_A and B data bundled
in the astropy-iers-data dependency rather than to IERS_B only when automatic ...
```

Reply:
Overhauled text